frontend/src/metabase/visualizations/components/ChartClickActions.jsx

```diff
@@ -212,7 +218,7 @@ class ChartClickActions extends Component {
           this.close();
         }}
         placement="bottom-start"
-        offset={[-8, 8]}
```

Comment:
I didn't manage to get horizontal offsets flipped correctly, so I decided to use the default one to make it look ok when flipped.